grid.html

```diff
@@ -276,3 +280,19 @@ <h2>Docker Selenium</h2>
   <h3>Prerequisite</h3>
   <p>The only requirement to run a grid is to have Docker installed and working.
   <a href="https://www.docker.com/products/docker-desktop">Install Docker</a>
+
+<h2>Troubleshooting</h2>
```

Comment:
Could you please put this before the Docker Selenium section? Also, this should be an `<h4>` instead of an `<h2>`.

Reply:
Done. Changed header type